cclient/request/get_transactions_to_approve.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "get_transactions_to_approve.h"
+
+get_transactions_to_approve_req_t* get_transactions_to_approve_req_new() {
+  get_transactions_to_approve_req_t* req =
+      (get_transactions_to_approve_req_t*)malloc(
+          sizeof(get_transactions_to_approve_req_t));
+  if (req == NULL) {
+      return NULL;
```

Comment:
Do you set up the commit hook?

Reply:
My commit hook will stopped by `cpplint`([issue  213](https://github.com/iotaledger/entangled/issues/213)), and I haven't solve this problem yet.
So I checked it with running `clang-format` in command line and I missed this one.   
Sorry!